index.html

```diff
@@ -176,7 +176,40 @@ <h5>LET'S CONNECT</h5>
   </div>
   <hr>
 </section>
-
+<!--  ______________________________Footer-FORM-section___________________________  -->
+<footer>
+  <section class="form-container">
+    <h6>Contact me</h6>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+    I'd love to help with it!</p>
+    <form action="https://formspree.io/f/moqzgjoa" method="post">
+      <!-- input name -->
+      <label for="name" id="inputName"></label>
+      <input name="name"
+        id="name"
+        type="text"
+        placeholder="Lorem Ipsum"
+        maxlength="30"
+        required>
+        <br>
+      <!-- input email -->
+      <label for="email"></label>
+      <input name="Email"
+        id="email"
+        type="email"
+        placeholder="loremipsum@gmail.com"
+        required>
+        <br>
+      <!-- input text area -->
+      <label for="text"></label>
+        <textarea name="textarea" id="inputTextArea" ols="50"rows="10" placeholder="Write your message here..."
+                  maxlength="500"
+                  required></textarea><br>
+      <!-- button -->
+      <button id="formButton" type="submit">Get in Touch</button>
```

Comment:
You're doing a fantastic job with your contact form! I just wanted to share a **couple of friendly suggestions to help make it even closer to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122):**

- [ ] It would be great if you could add the correct **background image to your design.** Currently, it seems to be missing the image. 🤓
- [ ] Could you please take a **look at the size and borders of your text area**? At the moment, it doesn't quite match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122). Adding a slight radius, as shown in the template, would be awesome. 🙋‍♂️
- [ ] Additionally, it appears that the `padding-top` in your contact form is a bit larger than in the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122). **It would be nice to adjust it to match the template more closely.** 😉

   ### Your design:
   ![image](https://github.com/fmalgarini/My-portfolio-Project/assets/107952724/79d8f356-87ea-43cf-9989-25c93afd7afa)


   ### Figma design:
   ![image](https://github.com/fmalgarini/My-portfolio-Project/assets/107952724/c77cd788-e504-461c-9db1-ae941463e334)